src/qml/NightstandPage.qml

```diff
@@ -0,0 +1,194 @@
+    }
+
+    ConfigurationValue {
+        id: nightstandEnabled
+
+        key: "/desktop/asteroid/nightstand/enabled"
+        defaultValue: false
+    }
+
+    PageHeader {
+        id: title
+        text: qsTrId("id-nightstand-page")
+    }
+
+    Flickable {
+        anchors.fill: parent
+        contentHeight: 2 * Dims.h(30) + 1 * Dims.h(34)
+        boundsBehavior: Flickable.DragOverBounds
+        flickableDirection: Flickable.VerticalFlick
+
+        GridLayout {
+            id: onOffSettings
+            columns: 2
+            anchors.fill: parent
+            anchors.margins: Dims.l(15)
+
+            Item {
+                id: brightnessSetting
+                height: Dims.h(30)
+                Layout.fillWidth: true
+                Layout.columnSpan: 2
+                Label {
+                    //% "Nightstand Brightness"
+                    text: qsTrId("id-nightstand-brightness")
+                    font.pixelSize: Dims.l(6)
+                    verticalAlignment: Text.AlignVCenter
+                    wrapMode: Text.Wrap
+                }
+
+                IconButton {
+                    iconName: "ios-remove-circle-outline"
+                    edge: undefinedEdge
+                    anchors.left: parent.left
+                    anchors.top: parent.top
+                    anchors.topMargin: Dims.h(10)
+                    onClicked: {
+                        var newVal = nightstandBrightness.value - 10
+                        if(newVal < 0) newVal = 0
+                        nightstandBrightness.value = newVal
+                    }
+                }
+
+                Label {
+                    text: nightstandBrightness.value + "%"
+                    font.pixelSize: Dims.l(6)
+                    horizontalAlignment: Text.AlignHCenter
+                    verticalAlignment: Text.AlignVCenter
+                    wrapMode: Text.Wrap
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    anchors.top: parent.top
+                    anchors.topMargin: Dims.h(10)
+                    height: Dims.h(20)
+                }
+
+                IconButton {
+                    width: Dims.w(20)
+                    height: width
+                    iconName: "ios-add-circle-outline"
+                    edge: undefinedEdge
+                    anchors.right: parent.right
+                    anchors.top: parent.top
+                    anchors.topMargin: Dims.h(10)
+                    onClicked: {
+                        var newVal = nightstandBrightness.value + 10
+                        if(newVal > 100) newVal = 100
+                        nightstandBrightness.value = newVal
+                    }
+                }
+            }
+
+            Item {
+                id: delaySetting
+                height: Dims.h(30)
+                Layout.fillWidth: true
+                Layout.columnSpan: 2
+                Label {
+                    //% "Nightstand delay"
```

Comment:
I think this is way too specific to be a setting we want to expose in asteroid-settings. Keep the ConfigurationValue implementation so that power users can override it from a terminal with dconf if they really want to but I don't think this delay belongs in the settings app.
On one hand I understand we are an OS that deals with nerds but on the other hand we still try to provide a somewhat usable smartwatch UX and I think that a key element is simplicity, it's sometimes a hard balance to find between nerdy settings and overly simple but I think the nightstand delay configuration is clearly over the top.
My understanding is that this delay is mostly a workaround for power pins bad contacts which cause connection/disconnection loops and I think this is the kind of settings where a sane default value of a couple of seconds is good for 99.9% of users and we don't need to clutter a default app to accommodate for the 0.1%

Reply:
The purpose is to allow the user to pick up the watch and bring it closer without it reverting to the regular watchface as described [here.](https://github.com/AsteroidOS/asteroid/issues/151#issuecomment-1204035843)